migrations/2023-10-19-102636_back_fill_n_remove_connector_response/up.sql

```diff
@@ -0,0 +1,11 @@
+-- Your SQL goes here
+ALTER TABLE payment_attempt 
+ADD COLUMN authentication_data JSON, 
+ADD COLUMN encoded_data TEXT;
+
+UPDATE payment_attempt 
```

Comment:
did you test this query?

Reply:
Yes, but we won't be running this directly on deployment environments to avoid load on DB.